From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: update required version of udev

The 071 release is needed to handle the input changes.  Older versions
will work properly with module-based systems, but not for users that
build input stuff into the kernel.


Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/Changes |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/Documentation/Changes
+++ gregkh-2.6/Documentation/Changes
@@ -65,7 +65,7 @@ o  isdn4k-utils           3.1pre1       
 o  nfs-utils              1.0.5                   # showmount --version
 o  procps                 3.2.0                   # ps --version
 o  oprofile               0.9                     # oprofiled --version
-o  udev                   058                     # udevinfo -V
+o  udev                   071                     # udevinfo -V
 
 Kernel compilation
 ==================
